superset/assets/src/SqlLab/constants.js

```diff
@@ -48,3 +48,13 @@ export const TIME_OPTIONS = [
 export const SQL_EDITOR_GUTTER_HEIGHT = 5;
 export const SQL_EDITOR_GUTTER_MARGIN = 3;
 export const SQL_TOOLBAR_HEIGHT = 51;
+
+// kilobyte storage
+export const KB_STORAGE = 1024;
+export const BYTES_PER_CHAR = 2;
+
+// browser's localStorage max usage constants
+export const LOCALSTORAGE_MAX_QUERY_AGE_MS = 24 * 60 * 60 * 1000; // 24 hours
+export const LOCALSTORAGE_MAX_USAGE_KB = 5 * 1024; // 5M
+export const LOCALSTORAGE_WARNING_THRESHOLD = 0.9;
+export const LOCALSTORAGE_WARNING_MESSAGE_THROTTLE_MS = 8000; // warning type toast duration
```

Comment:
for danger type toast, it has 8000 MS duration. if we set lower throttle threshold, user will see multiple message box, which is also annoying.
https://github.com/apache/incubator-superset/blob/1dd4d7a587a55525ff14e1e051d268adccbc15a1/superset/assets/src/messageToasts/actions/index.js#L77